pkg/cmd/kind/load/image-load/image-load.go

```diff
@@ -0,0 +1,305 @@
+		})
+	}
+	return errors.UntilErrorConcurrent(fns)
+}
+
+// TODO: we should consider having a cluster method to load images
+
+// loads an image tarball onto a node
+func loadImage(imageTarName string, node nodes.Node) error {
+	f, err := os.Open(imageTarName)
+	if err != nil {
+		return errors.Wrap(err, "failed to open image")
+	}
+	defer f.Close()
+	return nodeutils.LoadImageArchive(node, f)
+}
+
+// save saves images to dest
+func save(binary string, images []string, dest string) error {
+	commandArgs := append([]string{"save", "-o", dest}, images...)
+	return exec.Command(binary, commandArgs...).Run()
+}
+
+// imageID return the Id of the container image
+func imageID(binary, containerNameOrID string) (string, error) {
+	cmd := exec.Command(binary, "image", "inspect",
+		"-f", "{{ .Id }}",
+		containerNameOrID, // ... against the container
+	)
+	lines, err := exec.OutputLines(cmd)
+	if err != nil {
+		return "", err
+	}
+	if len(lines) != 1 {
+		return "", errors.Errorf("image ID should only be one line, got %d lines", len(lines))
+	}
+	return lines[0], nil
+}
+
+// removeDuplicates removes duplicates from a string slice
+func removeDuplicates(slice []string) []string {
+	result := []string{}
+	seenKeys := make(map[string]struct{})
+	for _, k := range slice {
+		if _, seen := seenKeys[k]; !seen {
+			result = append(result, k)
+			seenKeys[k] = struct{}{}
+		}
+	}
+	return result
+}
+
+// checkIfImageExists makes sure we only perform the reverse lookup of the ImageID to tag map
+func checkIfImageReTagRequired(node nodes.Node, imageID, imageName string, tagFetcher imageTagFetcher) (exists, reTagRequired bool, sanitizedImage string) {
+	tags, err := tagFetcher(node, imageID)
+	if len(tags) == 0 || err != nil {
+		exists = false
+		return
+	}
+	exists = true
+	sanitizedImage = sanitizeImage(imageName)
+	if ok := tags[sanitizedImage]; ok {
+		reTagRequired = false
+		return
+	}
+	reTagRequired = true
+	return
+}
+
+// sanitizeImage is a helper to return human readable image name
+// This is a modified version of the same function found under providers/podman/images.go
+func sanitizeImage(image string) (sanitizedName string) {
+	const (
+		defaultDomain    = "docker.io/"
+		officialRepoName = "library"
+	)
+	sanitizedName = image
+
+	if !strings.ContainsRune(image, '/') {
+		sanitizedName = officialRepoName + "/" + image
+	}
+
+	i := strings.IndexRune(sanitizedName, '/')
+	if i == -1 || (!strings.ContainsAny(sanitizedName[:i], ".:") && sanitizedName[:i] != "localhost") {
+		sanitizedName = defaultDomain + sanitizedName
+	}
+
+	i = strings.IndexRune(sanitizedName, ':')
+	if i == -1 {
+		sanitizedName += ":latest"
+	}
+
+	return
+}
+
+func GetDefaultRuntime() (string, error) {
+	if _, err := osexec.LookPath("docker"); err != nil {
+		if !syserrors.Is(err, osexec.ErrNotFound) {
+			return "", err
+		}
+
+		if _, err := osexec.LookPath("podman"); err != nil {
+			return "", err
+		}
+		return "podman", nil
```

Comment:
is this the difference?

Reply:
because this was not working before https://github.com/kubernetes-sigs/kind/issues/2760

Reply:
I tried to use docker-image in the form of soft links, and I succeeded.
```bash
❯ ln -s /usr/bin/podman /usr/bin/docker
❯ kind load docker-image --name kwok-m1 docker.io/kindest/node:v1.24.0
enabling experimental podman provider
Image: "docker.io/kindest/node:v1.24.0" with ID "a7d31d743aca47d4cdb00d6f61ba58b2423a1ead57dffa234f8d4f402f0a049d" not yet present on node "kwok-m1-control-plane", loading...
❯ podman exec a67d9f6f893b crictl images ls | grep "docker.io/kindest/node"
docker.io/kindest/node                     v1.24.0              a7d31d743aca4       915MB
docker.io/kindest/node                     v1.25.0              d3da246e125a0       875MB
```

Reply:
```
❯ cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.2 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.2 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

Reply:
something seems to have changed in podman or my memory is confused 😄 

@BenTheElder it seems we can have the common load-image with no effort